spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe User do
+  context '::authenticate' do
+    let!(:user) do
+      User.create(name: 'Katie', username: 'KS', email: 'K@email.com', password: 'secret', password_confirmation: 'secret')
```

Comment:
Metrics/LineLength: Line is too long. [123/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)